stl/inc/xiosbase

```diff
@@ -248,6 +248,11 @@ public:
         // merge in format flags argument
         const ios_base::fmtflags _Oldfmtflags = _Fmtfl;
         _Fmtfl |= _Newfmtflags & _Fmtmask;
+
+        _STL_ASSERT(__popcnt(_Fmtfl & basefield) < 2, "Multiple basefield flags set. Try setf(flags, mask) instead.");

```

Comment:
`__popcnt` intrinsic will emit `popcnt` instruction, without fallback path. We still support CPUs without this instruction by the STL, even though they may not be supported by the latest Windows.

To have `popcount` with fallback path and available in pre-C++20 mode, use `_Checked_popcount` from `<__msvc_bit_utils.hpp>`.